Neos.Flow/Classes/Mvc/Routing/Dto/RouteLifetime.php

```diff
@@ -0,0 +1,87 @@
+<?php
+namespace Neos\Flow\Mvc\Routing\Dto;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * RouteLifetime to be associated with matched/resolved routes
+ *
+ * RouteLifetime can be set by Route Part handlers via the ResolveResult/MatchResult return values
+ * If one or more lifetime is added the lifetime will be applied to to the corresponding cache entries.
+ *
+ * @Flow\Proxy(false)
+ */
+final class RouteLifetime
+{
+
+    /**
+     * @var int|null
+     */
+    private $value;
```

Comment:
Why `int|null`
I assume because `null` has some special meaning (does not expire?).
IMO we should make that more explicit or at least document it and add an example.

Reply:
Added a comment to explain null = undefined vs 0 = infinite

Reply:
I see, thanks.
I still think it's kind of weird to have a Value Object that can represent `undefined` and I would suggest to solve that in a higher level (i.e. either a route _has_ a lifetime or not).